resources/assets/components/TagsFilter/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { map } from 'lodash';
+import classnames from 'classnames';
+
+import '../Tags/tags.scss';
+
+class TagsFilter extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      'good-photo': false,
+      'good-quote': false,
+      'hide-in-gallery': false,
+      'good-for-sponsor': false,
+      'good-for-storytelling': false,
+    }
```

Comment:
should I be passing the state down  from `PostFilter` here instead of declaring again?